docs/design/lexical_conventions/words.md

```diff
@@ -96,12 +97,28 @@ The following words are interpreted as keywords:
 -   `where`
 -   `while`
 
+### Raw identifiers
+
+A raw identifier looks like `r#<identifier>`. This can be used for identifiers
+which have the same spelling as keywords; for example, `r#impl`. It can help
+when using C++ code with identifiers that are a keyword in Carbon.
```

Comment:
Maybe:
```suggestion
which have the same spelling as keywords; for example, `r#impl`. It can help
when using C++ code with an identifier that is a keyword in Carbon.
```

Or:
```suggestion
which have the same spelling as keywords; for example, `r#impl`. It can help
when using C++ code with identifiers that are keywords in Carbon.
```